3rdparty/stout-borrowed-ptr/repos.bzl

```diff
@@ -10,12 +10,17 @@
 ########################################################################
 
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
-
-load("//3rdparty/stout-stateful-tally:repos.bzl", stout_stateful_tally_repos="repos")
+load("//3rdparty/stout-stateful-tally:repos.bzl", stout_stateful_tally_repos = "repos")
 
 def repos(external = True, repo_mapping = {}):
+    """Add stout-stateful-tally repo.
```

Comment:
This comment is actually for the `repos()` function and should describe more generally that it "adds dependent repositories".